app/src/main/java/com/example/bookwormadventuresdeluxe2/Book.java

```diff
@@ -1,10 +1,5 @@
 package com.example.bookwormadventuresdeluxe2;
 
-/**

```

Comment:
why are you deleting the header comments?

Reply:
I had moved the comments down to just above the class (I tried to unify the style but it seems divided now that I look more) reverted to where it used to be and changed others to follow